analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/VersionConflictProjectFunctionalTest.java

```diff
@@ -127,7 +133,7 @@ public void validateTransitiveEnabledAndShadow() throws IOException, URISyntaxEx
             });
         });
 
-        assertTrue(systemOutRule.getLog().contains("Detected use of conflict resolution strategy strict"));
-        assertTrue(systemOutRule.getLog().contains("Passing 19 GAVs following into the REST client api"));
+        assertThat(systemOutRule.getLog()).contains("Detected use of conflict resolution strategy strict");
+        assertThat(systemOutRule.getLog()).contains("Passing 18 GAVs into the REST client api [");
```

Comment:
The new message returns one less (18 instead of 19) as the root project GAV is no longer passed to this endpoint.

Reply:
Understood, make sense.